circles/templates/circles/circle_form.html

```diff
@@ -24,11 +24,11 @@
                                placeholder="Circle name"
                                value="{{ circle.name }}"
                                required>
-                        <button class="btn btn-raised btn-material-light-blue" type="submit">Save</button>
-                        <a class="btn btn-raised btn-danger"
-                           href="{% url 'circles:delete' pk=circle.pk %}">Delete</a>
+                        <button class="btn btn-raised btn-primary pull-right" type="submit">Save</button>
```

Comment:
So. I've been taking a look, and for me it's ok to have the buttons for edit on the left hand side, but the others on the right.